### ISSUE TITLE
Registrar: enable removing clientHold status

### ISSUE THREAD

**Issue by vohmar** [internetee/registry#1481]
Related to #1428 1428

Registrar portal is lacking an option to remove clientHold status. 

there should be an option in registrar portal ui to remove clientHold status from a domain that has it set.

In addition there should be and example epp domain update request in the xml console  to remove clientHold status - this might be totally new sample request or added to the preexisting domain update request.

for example:
```
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<epp xmlns="https://epp.tld.ee/schema/epp-ee-1.0.xsd">
  <command>
    <update>
      <domain:update
       xmlns:domain="https://epp.tld.ee/schema/domain-eis-1.0.xsd">
        <domain:name>timo-4a.ee</domain:name>
        <domain:rem>
          <domain:status s="clientHold"/>
        </domain:rem>
      </domain:update>
    </update>
    <clTRID>timo-1579351654</clTRID>
  </command>
</epp>
```

**Comment by vohmar** [comment]
for xml console one could just update domain:update sample request like so:

```
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<epp xmlns="https://epp.tld.ee/schema/epp-ee-1.0.xsd">
  <command>
    <update>
      <domain:update
       xmlns:domain="https://epp.tld.ee/schema/domain-eis-1.0.xsd">
        <domain:name>example.ee</domain:name>
        <domain:add>
          <domain:ns>
            <domain:hostAttr>
              <domain:hostName>ns1.example.com</domain:hostName>
            </domain:hostAttr>
            <domain:hostAttr>
              <domain:hostName>ns2.example.com</domain:hostName>
            </domain:hostAttr>
          </domain:ns>
          <domain:contact type="tech">mak21</domain:contact>
        </domain:add>
        <domain:rem>
          <domain:ns>
            <domain:hostAttr>
              <domain:hostName>ns1.example.net</domain:hostName>
            </domain:hostAttr>
          </domain:ns>
          <domain:contact type="tech">mak21</domain:contact>
          <domain:status s="clientHold"/>
        </domain:rem>
        <domain:chg>
          <domain:registrant>mak21</domain:registrant>
          <domain:authInfo>
            <domain:pw>newpw</domain:pw>
          </domain:authInfo>
        </domain:chg>
      </domain:update>
    </update>
    <extension>
      <secDNS:update xmlns:secDNS="urn:ietf:params:xml:ns:secDNS-1.1">
        <secDNS:rem>
          <secDNS:keyData>
            <secDNS:flags>257</secDNS:flags>
            <secDNS:protocol>3</secDNS:protocol>
            <secDNS:alg>8</secDNS:alg>
            <secDNS:pubKey>700b97b591ed27ec2590d19f06f88bba700b97b591ed27ec2590d19f</secDNS:pubKey>
          </secDNS:keyData>
        </secDNS:rem>
      </secDNS:update>
      <eis:extdata xmlns:eis="https://epp.tld.ee/schema/eis-1.0.xsd">
        <eis:legalDocument type="pdf">
          dGVzdCBmYWlsCg==
        </eis:legalDocument>
      </eis:extdata>
    </extension>
    <clTRID>timo-1579814872</clTRID>
  </command>
</epp>
```

**Comment by yulgolem** [comment]
Blocked by https://github.com/internetee/docker-images/pull/5#issuecomment-579198534, https://github.com/internetee/registry/issues/846 (WIP)

**Comment by vohmar** [comment]
Looks good and everything seems to work as expected. 

Some suggestions for consideration:

* It might be better to move the remove clienthold button from the domain index view to the domain details view. This wouldn't affect the domain index view as there will be relatively few domains with this status
* rename the example clientHold status removal example request in xml console to 'rem clientHold" or 'remove clientHold' to make it clear what this request is about

**Comment by vohmar** [comment]
domain info view returns error:
ArgumentError in Registrar::Domains#info

![image](https://user-images.githubusercontent.com/7734747/74845160-45592c00-5337-11ea-89c3-630a67b980c1.png)

**Comment by vohmar** [comment]
fixed by #1533
